tools/video_tools/pyscenedetect_shot_detection.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="PySceneDetect_shot_detection" name="PySceneDetect" version="1.0.0">
+  <description>Runs shot detection on a video</description>
+    <requirements>
+    </requirements>
+  <command detect_errors="exit_code"> $__tool_directory__/pyscenedetect_shot_detection.py $input $threshold $amp_shots $frame_stats</command>
+  <inputs>
+	<param name="input" type="data" format="video" label="Video File" help="A video file input for shot detection"/>
```

Comment:
We are working on basic Video types in Galaxy. What datatypes do you need to support?

Reply:
Sounds exciting! So far we have extended a few data types for input media file (instead of using all as binary), such as Music, Speech. We may need to further refine the media type when use cases come up. What are your use cases for video?

Reply:
I really appreciate your review and suggestions! It's a pleasant surprise to receive your first review a few days ago. At first I thought you are one of the team members of our partner in Germany (we work with Fraunhofer), but then I see your profile in Github, and glad to find that you are one of the Galaxy developers! I mentioned you to my manager and we are very glad to have your input, and hope you will continue to do so!

By the way, if you are interested, here is a brief intro to our AMPPD project https://wiki.dlib.indiana.edu/pages/viewpage.action?pageId=531699941. You mentioned that you are involved with some non bioinfo Galaxy community, would you give me some pointer on that? We are very interested in extending our collaboration with other institutions, or at least be involved in the community. Thanks!

Reply:
> What are your use cases for video?

For example, in ecology, you also have audio from birds or videos from cells. Those can be analyzed and we tried to get the basic video and audio datatypes into Galaxy (the main source).

We wrote this over the weekend, maybe its useful: https://galaxyproject.eu/posts/2020/08/22/three-steps-to-galaxify-your-tool/

> I mentioned you to my manager and we are very glad to have your input, and hope you will continue to do so!

Cool, I will. Please also ping me if I miss something bigger.

> You mentioned that you are involved with some non bioinfo Galaxy community, would you give me some pointer on that? 

[Ecology](https://ecology.usegalaxy.eu/), Imaging, [Climate science](climate.usegalaxy.eu/), [Cheminformatic](cheminformatics.usegalaxy.eu/)

> https://wiki.dlib.indiana.edu/pages/viewpage.action?pageId=531699941

Cool thanks for sharing! Do you know Carrie (https://github.com/cganote)?

> We are very interested in extending our collaboration with other institutions or at least be involved in the community. 

If you like let's talk about all this :)

Reply:
Thank you very much for the feedback and links! I created AMP-860 to address the changes that apply to all existing MGM tools in AMP galaxy repo. I will merge this PR for now so that our staging env will be up-to-date with test env. All dev work will be PR'ed and merged to master branch first, then merge to staging branch when tested ready.